### PR TITLE
Add head seo

### PR DESCRIPTION
**Pull request by geositta**

### CODE REVIEW

**Comment by geositta** [comment]
Added unique keys since React requires this when elements such as meta tags are repeated.

**Comment by geositta** [comment]
[
Screen Shot 2022-04-07 at 10 49 22 PM
](url)

File: pages/index.js [hunk 1]
```diff
@@ -45,164 +46,170 @@ export default function Home(props) {
...
+      <Layout>
+        <Head>
+          <title>
+            Help UA Now | Donate to vetted, volunteer run fundraisers in
+            Ukraine.
+          </title>
+          <meta
+            key='home-description'
+            name="description"
+            content="“Help UA Now connects you to local, volunteer run fundraisers in Ukraine. Provide urgent, life-saving aid directly to those in need through one of our vetted partners. Donate to as many causes as you’d like, all in one place"
```
Comment (shenanigansd) [inline pages/index.js:58]: There are two quotes here, causing a quote to show on the actual embed

**Review by shenanigansd — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 1 above)
Comment (geositta) [reply pages/index.js]: I pushed a fix, good catch!

**Review by shenanigansd — DISMISSED** [review]

**Pull request merged**